README.md

```diff
@@ -1 +1,153 @@
-# Catalog-of-my-things
\ No newline at end of file
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About ](#-about-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage ](#usage-)
+    - [Test ](#test-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgement ](#acknowledgement)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 About <a name="about-the-project"></a>
+ _**Catalog of my things**_ built with [Ruby](https://www.ruby-lang.org/en/). In this project We have created:
+
+Catalog of my things Project is a Ruby-based console application designed to allows users to organize and store data about their personal catalog of items. With this application, you can easily manage various types of collections, such as books, music albums, movies, or any other items you wish to catalog.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Item Management: Create and view and delete items in your catalog.
+- Data Persistence
+- User-Friendly Interface
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running. To get more information, read the [installation instructions](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md).
+- You should be able to use [IRB](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Features).
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:iamsjunaid/catalog-of-my-things.git
+  cd catalog-of-my-things
+```
+
+### Install
+
+Install project dependencies with:
+
+```
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  ruby main.rb
+```
+
+### Test <a name="test"></a>
+
+To run the tests, execute the following command:
+
+```
+  bundle exec rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **Junaid Syed**
+
+- GitHub: [@iamsjunaid](https://github.com/iamsjunaid)
+- LinkedIn: [@JunaidAhmedSyed](https://www.linkedin.com/in/junaidahmedsyed/)
```

Comment:
Thanks @abdulhamiid, it's true, the profile is not available.
This is because @iamsjunaid has issues with his LinkedIn profile.
The link was replaced with this 👉 [commit](https://github.com/iamsjunaid/Catalog-of-my-things/pull/51/commits/5d72e57dff46a70d4e6a6abe779bbeee10a8b857)